getting-started/ci-setup/jenkins.md

```
@@ -78,9 +78,9 @@ pipeline {
 ```
 {% endcode %}
 
-### Using last failed flag with shards and orchestration
+### Using `--last-failed` with shards and orchestration
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)